### ISSUE TITLE
[Sign up. UBS] The frame line of the 'Password' and 'Confirm password' fields is not displayed correctly

### ISSUE THREAD

**Issue by Petryulya** [ita-social-projects/GreenCity#3949]
**Environment:** Windows 10, only on browser Opera (version: 84.0.4316.42).
**Environment:** always
**Build found:** 31.03.22

Steps to reproduce
1. Go to 'https://ita-social-projects.github.io/GreenCityClient/#/ubs'
2. Click on 'Sign up'
3. Enter Password which contain at least one character of Uppercase letter (A-Z), Lowercase letter (a-z), Digit (0-9), Special character (/*...), e.g.: QQQqqq1/
4. Enter 'Confirm password' that is not the same as the 'Password'
5. Click on 'Password' field
6. Click on 'Confirm password' field again
7. Pay attention to the frame line of the field (see attached file)

**Actual result**
The bottom line of the frame in the 'Password' and 'Confirm password' fields is missing.

**Expected result**
The bottom line of the frame in the 'Password' and 'Confirm password' fields is not missing.

**User story**
 #2929
![5](https://user-images.githubusercontent.com/102656931/161094051-95a49631-bbc2-4fc7-bc3e-a29e21cd766a.jpg)

https://user-images.githubusercontent.com/102656931/161094214-b42c14fb-c9db-49d2-824a-6edf477d3e57.mp4

**Comment by voronkooleksandr** [comment]
This bug does not reproduce on the developer's browser
